test/binCases/config-location/webpackfile-config/packages/foo/webpackfile.js

```diff
@@ -0,0 +1,7 @@
+var path = require("path");
+
+module.exports = [
```

Comment:
At this level, add a `webpack.config.js` file as well, it shouldn't break the test if implemented correctly.

Reply:
If I add a `webpack.config.js` in the same level as `webpackfile.js` it will break the test because if they are on the same level then `webpack.config.js` will be used. I've added a `webpack.config.js` two level above: https://github.com/webpack/webpack-cli/pull/693/files#diff-3f5d33891236c7320bd882814444bca0R1